packages/addons-v5/test/unit/commands/addons/info.unit.test.js

```diff
@@ -108,9 +110,10 @@ State:        created
           `=== www-db
 Attachments:  acme-inc-www::DATABASE
 Installed at: Invalid Date
+Max Price:    $5/month
 Owning app:   acme-inc-www
 Plan:         heroku-postgresql:mini
-Price:        $5/month
+Price:        ~$0.007/hour
```

Comment:
3 decimal places was probably a mistake, but not our call. 3 significant figures would likely be better.

Reply:
There were some lengthy discussions about this and it was decided to use three decimal places.